jdtls.ext/com.microsoft.java.maven.plugin/src/main/java/com/microsoft/java/maven/ArtifactResult.java

```diff
@@ -0,0 +1,105 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Microsoft Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Microsoft Corporation - initial API and implementation
+ *******************************************************************************/
+
+package com.microsoft.java.maven;
+
+public class ArtifactResult {
+
+    private String groupId;
+
+    private String artifactId;
+
+    private String version;
+
+    private String className;
+
+    private String fullClassName;
+
+    private int usage;
+
+    private int kind;
+
+    public static final int PREFIX = 1;
```

Comment:
usage of these constants?

Reply:
to represent whether the artifactResult is a prefix-match result or a fuzzy one, and it is used when sort the artifactResults.